src/System.Console/pkg/unix/System.Console.pkgproj

```diff
@@ -7,9 +7,9 @@
     <PreventImplementationReference>true</PreventImplementationReference>
   </PropertyGroup>
   <ItemGroup>
-    <ProjectReference Include="..\..\src\System.Console.builds" >
+    <ProjectReference Include="..\..\src\System.Console.csproj" >
       <!-- the binary is not specific to Linux, but we only do a Linux & OSX pass, not Unix-specific pass -->
```

Comment:
I agree we should keep this PR scoped we can do the larger Unix conversion at a later time.